docs/versioning-policy.md

```diff
@@ -0,0 +1,98 @@
+# Tobira Releases and Versioning Policy
+
+Tobira is released via [GitHub releases](https://github.com/elan-ev/tobira/releases).
+Every release contains a pre-built binary for Linux-systems and a full changelog.
+
+We currently only maintain one branch and thus only have one release track.
+In other words: we don't backport bugfixes or other patches.
+Only the latest version is officially supported.
+
+Tobira version identifiers look like `x.y-name`, e.g. `3.12-cherry`.
+
+- `x`: main version number. Is increased for releases that contain new features, major UI changes, or any other non-small change.
+- `y`: patch version number. Is increased for releases that only contain bugfixes and other very minor changes. Reset to 0 whenever `x` is increased.
+- `name`: an arbitrary nickname for this release. Is associated to `x` and changes if and only if `x` changes.
+
+**Important**: the version identifier carries no semantic information you can rely on!
+It is simply an identifier, that's easy to communicate and useful for "marketing".
+The rules for increasing the numbers are deliberately very underspecified and  just very roughly approximate the user-perceived "size" of the release.
+You must **not** rely on these numbers to derive any information about breaking changes or ease of updating.
+**This is not SemVer!**
```

Comment:
Unnecessary repetition.

```suggestion
```

Reply:
I disagree. As this is a document for humans, stating something explicitly that can already be inferred from previous information is not always a bad idea. And I think in this case it is a good idea, because the version number might easily be interpreted this way. I can try to shorten this section though, if that's an acceptable compromise for you?